internal/ot/README.md

```diff
@@ -1,9 +1,19 @@
 # Oblivious Transfer (OT)
-
-# protocol
 Oblivious transfer is a cryptographic primitive crucial to building secure multiparty computation protocols. A secure OT protocol allows for two untrusted parties, a sender and a receiver, to perform data exchange in the following way. A sender has as input two messages _M<sub>0</sub>_, _M<sub>1</sub>_, and a receiver has a selection bit _b_. After the OT protocol, the receiver will learn only the message _M<sub>b</sub>_ and not _M<sub>1-b</sub>_, while the sender does not learn the selection bit _b_. This way the receiver does not learn the unintended message (protect against malicious receiver), and the sender cannot forge messages since he does not know which message will be learnt by the receiver (protect against malicious sender).
-After 40 years since its invention, two notable base OT protocols are the Naor-Pinkas OT[1] and the Simplest Protocol for OT[2]. An OT extension uses base OT to exchange short messages in order to accomplish much larger number of effective OT. One of the most efficient OT extension is the IKNP OT extension [3]. Recently, [4] improved the IKNP OT extension protocol by using examining the protocol in a code theoretic lense, and extended IKNP to KKRT 1 out of n OT extension. The aforementioned base OT protocols and OT extension protocols are all implemented in this package.
-Discalimer: This is a work in progress.
+After 40 years since its invention, two notable base OT protocols are the Naor-Pinkas OT[1] and the Simplest Protocol for OT[2]. An OT extension uses base OT to exchange short messages in order to accomplish much larger number of effective OT. One of the most efficient OT extension is the IKNP OT extension [3]. Recently, [4] improved the IKNP OT extension protocol by using examining the protocol in a code theoretic lense, and extended IKNP to KKRT 1 out of n OT extension.
```

Comment:
```suggestion
After 40 years since its invention, two notable base OT protocols are the Naor-Pinkas OT[1] and the Simplest Protocol for OT[2]. An OT extension uses a base OT to exchange short messages in order to achieve many more effective OTs. One of the most efficient OT extensions is the IKNP OT extension [3]. Recently, [4] improved the IKNP OT extension protocol by examining the protocol in a code theoretic lens and extended IKNP into the KKRT, 1 out of n, OT extension.
```